Task5/listOfProducts.txt

```diff
@@ -0,0 +1,3 @@
+Name: meat, price: 100$, amount: 10, id: 2
+Name: grapes, price: 60$, amount: 15, id: 3
```

Comment:
I check your commit log in task5 branch and looks like you have there commit add task4.
Try to use **git revert --strategy resolve <commit-id>** to remove add task4 commit. Then make commit and push and Task4 folder should go from Task5 PR.